client/src/sagas/index.js

```diff
@@ -1,6 +1,13 @@
 import { all } from 'redux-saga/effects';
-import { watchLoginAsync, watchRegisterAsync } from './auth';
+
+import { watchLoginAsync, watchLogout, watchRegisterAsync } from './auth';
+import { watchAddAlert } from './alert';
 
 export default function* rootSaga() {
-  yield all([watchLoginAsync(), watchRegisterAsync()]);
+  yield all([
+    watchLoginAsync(),
+    watchLogout(),
+    watchRegisterAsync(),
+    watchAddAlert(),
+  ]);
```

Comment:
💍 Same question here: will we end up having one "root saga" to rule them all?

Reply:
Yes. Basically the entry point for the state management system.